### PR TITLE
feat: UTP-1543 Rule history log

### PR DESCRIPTION
**Pull request by shenbagavallisg**

### CODE REVIEW

File: ui/react/pages/RuleProfile/Edit/index.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ import Destination from '../Tabs/Destination';
 import Split from '../Tabs/Split';
 import Limit from '../Tabs/Limit';
 import * as actions from '../actions';
+import HistoryLog from 'ut-audit/modules/history/ui/react/containers/HistoryLog';
```
[the remark this answers is not in the document]
Comment (ranjith-sg) [reply ui/react/pages/RuleProfile/Edit/index.js]: @kalinkrustev , I have moved the history log container to ut-front-react/ containers (looks like Document container), its not yet ready for use. Shall I keep the history log container in ut-front-react or will move it to ut-history ?

**Comment by ranjith-sg** [comment]
@shenbagavallisg  now history log is available at 'ut-history/ui' please refer it

File: ui/react/pages/RuleProfile/Edit/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (shenbagavallisg) [reply ui/react/pages/RuleProfile/Edit/index.js]: @kalinkrustev, Now I have referred the history log from 'ut-history/ui'. Please merge the request if the changes are fine.

File: history/helpers.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
+let { splitTags } = require('../config');
+
+const propMap = {
```
Comment (stambata) [inline history/helpers.js]: Isn't it somehow possible to reuse [this](https://github.com/softwaregroup-bg/ut-rule/blob/a64ed0a7dd784217e5c2d1cae8e8e1fee7c78ac9/ui/react/pages/RuleProfile/helpers.js#L219) code? Duplicating 150 lines of code with just slight differences doesn't look like a good idea.

Comment (ranjith-sg) [reply history/helpers.js]: Yes similar type of code exists in front end code with some differences, this above one is used at back end.  I will make it an a common place and refer  it at required places

Comment (ranjith-sg) [reply history/helpers.js]: @stambata  Can you have look at now ?

**Pull request merged**